eng/pipelines/templates/checkout-job.yml

```diff
@@ -5,13 +5,13 @@
 ### and OSX build clients don't have 7-zip installed) and simplify other details such
 ### as line endings.
 jobs:
-- job: 'checkout_windows'
-  displayName: 'Checkout (Windows)'
+- job: checkout
+  displayName: Checkout
 
   pool:
     # Public Windows Build Pool
     ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      vmImage: 'windows-latest'
```

Comment:
Nit - do we need to use the Windows variant for the checkout? The Linux one seems to be approximately twice as fast.

Reply:
No, it shouldn't matter what OS to run the bundle create command. I will switch, thanks